couler/core/templates/output.py

```diff
@@ -145,12 +148,15 @@ def _container_output(step_name, template_name, output):
                         template_name,
                         output_id,
                     )
+                out_type = 'io' if len(o) == 2 else ''
```

Comment:
Needed to know the artifact type for later decisions.
Maybe the same needed for other types (oss, s3)...